.github/workflows/publish.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
     - 'main'
     - 'dev'
+    - 'devops/add-docker-sec'
```

Comment:
is this needed?

Reply:
actually not! removed it in https://github.com/babylonlabs-io/babylon/pull/264/commits/b627a675654f77b80a183e87c31cdbcee597d993